### ISSUE TITLE
s build -d. ==> error: unknown option '-d'

### ISSUE THREAD

**Issue by newbaluku** [Serverless-Devs/Serverless-Devs#272]
I'm trying to use Nodejs12 in custom runtime. So I referred to the article @ https://www.alibabacloud.com/help/zh/doc-detail/311157.htm

have dowloaded Nodejs12 and unzip but encountered error when executing this step:

s build -d

error: unknown option '-d'

any direction where I have gone wrong?

**Comment by newbaluku** [comment]
Thanks for the quick reply. I found the problem.

it's because my yaml file name is not the default one, ie. s.yml / s.yaml

once I reverted back to the default name it works.

or alternatively add in the template filename when running build, ie. s build -d -t s-myfile.yml

perhaps the tool should provides a more helpful error description like, "template/yml file not found" or something of that sort instead of:

error: unknown option '-d'
